CSS list-style shorthand.

estelle opened the pull request:
Part of https://github.com/openwebdocs/project/issues/181

* added see also 
* added actual values to the list of values
* moved accessibility down to below examples
* reduced the explanation of an a11y example as the alternative method is the preferred method.
* links in a11y section

estelle committed:
CSS list-style shorthand.

Part of https://github.com/openwebdocs/project/issues/181

estelle committed:
typo

estelle committed:
removed "note"

chrisdavidmills commented on files/en-us/web/css/list-style/index.md:
````diff
@@ -175,6 +108,40 @@ List 2
+ul li::before {
+  content: "+ ";
+}
+```
````
The visual impact here is potentially quite a big issue. 

Question - does something like

```css
li::marker {
  color: transparent;
}
```

hide the bullets without hiding the list from Safari's accessibility tree?

chrisdavidmills reviewed (changes requested):
@estelle more great work, thanks! A few comments for you to mull over.

estelle committed:
Apply suggestions from code review

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

estelle commented on files/en-us/web/css/list-style/index.md:
```diff
@@ -50,86 +50,19 @@ list-style: revert-layer;
 
 - {{cssxref("list-style-type")}}
-  - : See {{cssxref("list-style-type")}}.
+  - : A `<counter-style>`, {{cssxref("string")}}, or `none`. If omitted in the shorthand, the default `disc` value is used. See {{cssxref("list-style-type")}}.
```
This is a shorthand property, so the links are just to the long-hand values that make up the shorthand and "none", which maybe shouldn't be there. It, like `symbols()` is a list-style-type value, not a value of list style>

Wondering if I should delete none?

chrisdavidmills replied:
Yeah, I think it is better to leave it, for the reasons you state.

estelle committed:
addressed feedback

estelle committed:
nav and a11y

estelle commented:
@chrisdavidmills Thank you for the review. I addressed the issues and questions you found. thank you. Ready for a re-review with one question: Should we remove none?

chrisdavidmills reviewed (approved):
@estelle lovely; all comments addressed, and this page is now looking shipshape. Thanks!
